src/Command.ts

```diff
@@ -334,6 +334,15 @@ export default abstract class Command {
   }
 
   protected handleRejectedODataPromise(res: any): void {
+    if (cli.optionsFromArgs?.options.debug) {
+      try {
+        throw new Error(res);
```

Comment:
Instead of throwing and catching an error, have you tried using:

```js
var error = new Error();
cli.error(error.stack);
```

It would be cleaner and if it does the same, I suggest that we use it instead.